examples/simple_dff/test_dff.py

```diff
@@ -8,22 +8,36 @@
 import cocotb
 from cocotb.clock import Clock
 from cocotb.runner import get_runner
-from cocotb.triggers import FallingEdge
+from cocotb.triggers import RisingEdge
+from cocotb.types import LogicArray
 
 
 @cocotb.test()
 async def dff_simple_test(dut):
     """Test that d propagates to q"""
 
+    # Assert initial output is unknown
+    assert LogicArray(dut.q.value) == LogicArray("X")
+    # Set initial input value to prevent it from floating
+    dut.d.setimmediatevalue(0)
```

Comment:
@ktbarrett: Is it worth a comment on `setimmediatevalue(0)` vs. `value = 0` ? I don't think the difference matters here anymore, now that it is set before the first `RisingEdge`, so this could use `value = 0` for simplicity.

Reply:
Yeah the difference doesn't matter anymore. I might prefer using `signal.value = 0` in that case to keep it simple.